tests/queries/0_stateless/02454_ReplacingMergeTreeNewEngine.reference

```diff
@@ -0,0 +1,64 @@
+d1	5	0
```

Comment:
Would it be better to choose a different name than *New* for this, since it's going to be "old" in a year? :)  How about ReplacingMergeTreeWithGC?